src/Marten.CommandLine/Marten.CommandLine.csproj

```diff
@@ -2,7 +2,7 @@
 
     <PropertyGroup>
         <Description>Command line tooling for Marten</Description>
-        <TargetFramework>netstandard2.1</TargetFramework>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Any specific reason why the target framework changed from `netstandard2.1` to `netcoreapp3.1`?

Reply:
The targeting of `Oakton.AspNetCore` changed to `netcoreapp3.1` and `net5.0` only.

Reply:
Thanks for the clarification @Hawxy 👍